src/early-late-bound.md

```diff
@@ -0,0 +1,122 @@
+# Early and Late Bound Variables
+
+A generic type may be [_universally_ or _existentially_ quantified][quant]. For
```

Comment:
I'm not sure this is true. All the generics in Rust correspond to universal quantification.